.github/actions/checkout-submodules-and-bootstrap/action.yaml

```diff
@@ -26,6 +30,9 @@ runs:
       uses: ./.github/actions/bootstrap
       with:
         platform: ${{ inputs.platform }}
+    - name: Dump disk info after checkout submodule & Bootstrap
+      shell: bash
+      run: bash scripts/dump_diskspace_info.sh
```

Comment:
if we already have a shell, you should not need to say `bash` here. Maybe make the `sh` executable and add `#!/usr/bin/bash` or similar?